USFMConverter/Core/Util/FileSystem.cs

```diff
@@ -131,5 +134,59 @@ Action<double> progressCallback
 
             return usfmDoc;
         }
+
+        public static Setting? LoadOptionConfig(string OutputFileFormat)
+        {
+            string path = $"appsettings_{OutputFileFormat}.json";
+            return File.Exists(path) ? JsonConvert.DeserializeObject<Setting>(File.ReadAllText(path)) : null;
+        }
+
+        public static void SaveOptionConfig(ViewData? dataContext)
+        {
+            Setting setting = new (dataContext);
+            string path = $"appsettings_{dataContext?.OutputFileFormat.Tag}.json";
+
+            // If file doesn't exist, create the file
+            if (!File.Exists(path))
+            {
+                File.WriteAllText(path, "{}");
+            }
+            
+            File.WriteAllText(path, JsonConvert.SerializeObject(setting, Formatting.Indented));
+        }
+
+        public static string LoadLastUsedFormat()
+        {
+            string path = "appsettings_format.json";
+            string lastUsedFormat = "";
+
+            if (!File.Exists(path))
+            {
+                string content = "{\"LastUsedFormat\": \"\"}";
+                File.WriteAllText(path, content);
+            }
+            else
+            {
+                string jsonFile = File.ReadAllText(path);
+                JObject jsonObj = JObject.Parse(jsonFile);
+
+                lastUsedFormat = (string) jsonObj["LastUsedFormat"];
+            }
+
+            return lastUsedFormat;
+        }
+
+        public static void SaveLastUsedFormat(ViewData? dataContext)
+        {
+            string path = "appsettings_format.json";
+            string lastUsedFormat = dataContext.OutputFileFormat.Tag.ToString();
+            
+            string jsonFile = File.ReadAllText(path);
+            JObject jsonObj = JObject.Parse(jsonFile);
+
+            jsonObj["LastUsedFormat"] = lastUsedFormat;
+            
+            File.WriteAllText(path, JsonConvert.SerializeObject(jsonObj, Formatting.Indented));
+        }
```

Comment:
Could these functions belong to a SettingManager class or something? 
They can share a constant field template for file name:
e.g. `private const string SETTING_FILE_TEMPLATE="appsettings_{0}.json"`